arlo-client/src/components/CreateAudit.tsx

```diff
@@ -89,6 +103,33 @@ const CreateAudit = ({ history }: RouteComponentProps<ICreateAuditParams>) => {
             Log in as a Jurisdiction Admin
           </Button>
         </>
+      ) : (
+        <>
+          {meta!.organizations.length > 0 &&
+            meta!.organizations.map(o =>
+              o.elections.map(election => (
+                <AuditLink
+                  to={`/election/${election.id}`}
+                  key={election.id}
+                  className="bp3-button bp3-intent-primary"
+                >
+                  {election.name || 'Not named yet'}
+                  {election.state && ` (${election.state})`}
+                </AuditLink>
+              ))
+            )}
+          {meta!.jurisdictions.length > 0 &&
+            meta!.jurisdictions.map(({ election }) => (
+              <AuditLink
+                to={`/election/${election.id}`}
+                key={election.id}
+                className="bp3-button bp3-intent-primary"
+              >
+                {election.name || 'Not named yet'}
```

Comment:
```suggestion
                {election.name ?? 'Not named yet'}
```

Reply:
@eventualbuddha I see nullish coalescing coming in TS 3.7, but we're still on 3.5 afaik. Also, I don't think this is a good use of it, specifically, since we would rather have `'Not named yet'` than an empty string, for example. Unless I'm missing something here.

Reply:
You're right, I'd misremembered that I'd updated Arlo to 3.7. That should be pretty low-hanging fruit now that `react-scripts` is updated. And in this particular case where we expect `election.name` might be an empty string then `||` makes more sense. In general I'd strive to use `??` instead and ensure the LHS, if defined, has a value we want to use.